contents/tutorials/calculating-new-returning-users.md

```diff
@@ -0,0 +1,93 @@
+---
+title: How to calculate new and returning users in PostHog
+date: 2023-04-25
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+featuredImage: ../images/tutorials/banners/tutorial-18.png
+tags: ["trends", "cohorts"]
+---
+
+Understanding user growth is critical to a successful product. A lack of new users or many users failing to return is a bad sign. This tutorial goes over the different ways to calculate new and returning users in PostHog, as well as insights you can create using those calculations.
+
+> **How does Google Analytics calculate new vs returning users?** When you first visit a site, Google Analytics generates a client ID for your device and sets it in your cookies. GA uses this client ID cookie to calculate if a user is new (didn’t have cookie set) or returning (had cookie set). This, like any tracking, isn’t 100% accurate.
+
+## Calculating new users
+
+There are multiple ways to calculate new users in PostHog depending on your situation and need.
+
+### Completed event for the first time cohort
+
+The first and easiest is creating a cohort where the users completed an event (such as pageviews, identify, custom sign up) for the first time. To set this up, go to the cohort tab, click new cohort, enter a title and description, then choose "completed an event for the first time" with your event and recency. 
```

Comment:
> (such as pageviews, identify, custom sign up)

Not sure I understand what is identify or custom sign up. Perhaps worth hyperlinking?